.github/workflows/publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: publish
+
+on:
+  push:
+    branches: ["develop", "main"]
```

Comment:
I think once we have a first release, this job should be triggered only when pushing to `main`. When is the best time to do this?